src/project-3.js

```diff
@@ -1,72 +1,114 @@
 // Do not change any of the function names
 
 const makeCat = (name, age) => {
+  const Cat = {
+    name,
+    age,
+  };
+  Cat.meow = function () {
+    return ('Meow!');
+  };
+  return Cat;
+};
   // create a new object with a name property with the value set to the name argument
   // add an age property to the object with the value set to the age argument
   // add a method called meow that returns the string 'Meow!'
   // return the object
-};
 
 const addProperty = (object, property) => {
   // add the property to the object with a value of null
   // return the object
   // note: the property name is NOT 'property'.  The name is the value of the argument called property (a string)
+  object[property] = null;
+  return object;
 };
 
 const invokeMethod = (object, method) => {
   // method is a string that contains the name of a method on the object
   // invoke this method
   // nothing needs to be returned
+  object[method]();
 };
 
 const multiplyMysteryNumberByFive = (mysteryNumberObject) => {
   // mysteryNumberObject has a property called mysteryNumber
   // multiply the mysteryNumber property by 5 and return the product
+  return mysteryNumberObject.mysteryNumber * 5;
 };
 
 const deleteProperty = (object, property) => {
   // remove the property from the object
   // return the object
+  delete object[property];
+  return object;
 };
 
 const newUser = (name, email, password) => {
   // create a new object with properties matching the arguments passed in.
   // return the new object
+  const User = {
+    name,
+    email,
+    password,
+  };
+  return User;
 };
 
 const hasEmail = (user) => {
   // return true if the user has a value for the property 'email'
   // otherwise return false
+  if (user.email) {
+    return true;
+  } return false;
 };
 
 const hasProperty = (object, property) => {
   // return true if the object has the value of the property argument
-  // property is a string
-  // otherwise return false
+  if (property === object.property) {
+    return true;
+  }
 };
 
 const verifyPassword = (user, password) => {
+  const User = {
+    user,
+    password,
+  };
+  if (password === user.password) {
```

Comment:
There is no `user.password` property. It should be `User.user` and `User.password` if you're selecting the User's properties. So if you wanted to compare the password coming in, and the User's password, you'd have to do it like `if (password === User.password)`.